vendor/sources/main.tf

```diff
@@ -1,3 +1,9 @@
+# Configure the Azure Provider
+provider "azurerm" {
+  version = "~> 2.32.0"
+  features {}
```

Comment:
features {} is this intended?

Reply:
Yes, without this entry the `provider` doesn't work. It's a required parameter since azurerm version 2:
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#argument-reference